CryptoBinPkg/Driver/readme.md

```diff
@@ -1,54 +1,99 @@
-# Crypto Driver
+# Shared Crypto Binaries
 

```

Comment:
Can we add the rules for <Module_Type>.Binary in this readme? Or, is this something that should be part of another dsc.inc since it will presumably be the same between platforms?

Reply:
It should be here. There's tradeoffs between providing an include file and a user manually having to understand what they're doing in greater detail. In this case, I think the binary rules are straightforward enough and can be managed in a single include that it is worth providing a file. I will add the file and update the instructions for it.